packages/design/src/colors.css

```diff
@@ -1,12 +1,73 @@
 :root {
+  /* Jobber Semantic Layer */
+
+  /* Interactive and States */
+  --color-interactive: var(--color-green);
+  --color-interactive--hover: var(--color-green--dark);
+  --color-destructive: var(--color-red);
+  --color-destructive--hover: var(--color-red--dark);
+  --color-cancel: var(--color-greyBlue);
+  --color-cancel--hover: var(--color-greyBlue--dark);
+
+  --color-disabled: var(--color-grey);
+  --color-disabled--secondary: var(--color-grey--lighter);
+
+  --color-focus: var(--color-yellow--light);
+
+  /* Status Colours */
+
+  --color-critical: var(--color-red);
+  --color-critical--light: var(--color-red--lightest);
+  --color-critical--dark: var(--color-red--dark);
+
+  --color-warning: var(--color-yellow);
+  --color-warning--light: var(--color-yellow--lightest);
+  --color-warning--dark: var(--color-yellow--dark);
+
+  --color-success: var(--color-green);
+  --color-success--light: var(--color-green--lightest);
+  --color-success--dark: var(--color-green--dark);
+
+  --color-informative: var(--color-lightBlue);
+  --color-informative--light: var(--color-lightBlue--lightest);
+  --color-informative--dark: var(--color-lightBlue--dark);
+
+  --color-inactive: var(--color-blue--light);
+  --color-inactive--light: var(--color-blue--lightest);
+  --color-inactive--dark: var(--color-blue--dark);
+
+  /* Typography */
+  --color-heading: var(--color-blue);
+  --color-text: var(--color-greyBlue--dark);
+  --color-text--secondary: var(--color-greyBlue);
+
+  /* Surfaces and Borders */
+  --color-surface: var(--color-white);
+  --color-surface--hover: var(--color-yellow--lightest);
+  --color-surface--background: var(--color-grey--lightest);
+  --color-surface--reverse: var(--color-greyBlue--dark);
+  --color-surface--success: var(--color-green--lighter);
+  --color-surface--error: var(--color-red--lighter);
+  --color-surface--warning: var(--color-yellow--lighter);
+  --color-surface--notice: var(--color-lightBlue--lightest);
+
+  --color-border: var(--color-grey--lighter);
+  --color-border--section: var(--color-blue);
+
+  /* Brand */
+  --color-brand: var(--color-green);
+  --color-brand--highlight: rgb(160, 215, 42);
```

Comment:
new color?

Reply:
yes, maps to brand highlight color that we currently use a gross version of green for:
![image](https://user-images.githubusercontent.com/39704901/115906505-d557a180-a424-11eb-8bc3-95471e14e22d.png)